### ISSUE TITLE
SqlException: Incorrect syntax near the keyword 'precision'.

### ISSUE THREAD

**Issue by MarelisCZ** [OpenDBDiff/OpenDBDiff#121]
Error information
Error Loading Server: 192.168.2.10, Database: pm_trend (while Reading views...)
SqlException: Incorrect syntax near the keyword 'precision'.
   v System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   v System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   v System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   v System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   v System.Data.SqlClient.SqlDataReader.TryConsumeMetaData()
   v System.Data.SqlClient.SqlDataReader.get_MetaData()
   v System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption, Boolean shouldCacheForAlwaysEncrypted)
   v System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)
   v System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)
   v System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)
   v System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior, String method)
   v System.Data.SqlClient.SqlCommand.ExecuteReader()
   v OpenDBDiff.Schema.SQLServer.Generates.Generates.GeneratePartitionFunctions.Fill(Database database, String connectioString) v C:\projects\opendbdiff\OpenDBDiff.Schema.SQLServer.Generates\Generates\GeneratePartitionFunctions.cs:řádek 42
   v OpenDBDiff.Schema.SQLServer.Generates.Generates.Generate.Process() v C:\projects\opendbdiff\OpenDBDiff.Schema.SQLServer.Generates\Generates\Generate.cs:řádek 114
   v OpenDBDiff.Schema.SQLServer.Generates.Front.SQLServerGenerator.Process() v C:\projects\opendbdiff\OpenDBDiff.Schema.SQLServer2005\Front\SQLServerGenerator.cs:řádek 36
   v OpenDBDiff.Front.ProgressForm.ProgressForm_Activated(Object sender, EventArgs e) v C:\projects\opendbdiff\OpenDBDiff\Front\ProgressForm.cs:řádek 65

3EF9D84079BB28CCBF0B50C77B390F11

**Comment by mbalatha** [comment]
Yes, we are also getting this error whenever a table column name is also a Sql Server keyword. All we need to do is wrap column names in `[]` square brackets or if it is a keyword then only wrap them in square brackets.
